fake_dependencies/capability-service/capability-data.json

```diff
@@ -9,7 +9,8 @@
           }
         ],
         "contexts": [],
-        "Topics": []
+        "Topics": [],
+        "RootId": "foo-5234"

```

Comment:
We need to replicate a proper capabiltyRootId logic. This is just a fake one.

Reply:
The logic for generating a RootId for a Capability is as following (**AT CURRENT DATE: 26.06.2019**):


The Capability ID is used, and only a maximum of 8 characters is kept, starting from the beginning of the ID.
Then, depending on how many characters the Capability name is, if it's more than 20 characters, it will take the name and only keep the first 20 characters. If it's not, then it'll just be used as is. Afterwards it will be combined with the previous mentioned id and converted to lowercase invariant.

It will look something like this: "name-id". The '-' is put in when combining the two variables.
Examples:
```
Name: ThisIsAPrettyLongNameIguess
Id: d32ec449-4cfb-4168-9671-9544c40f459c

Generated RootId: thisisaprettylongnam-d32ec449
```

```
Name: HelloPelle
Id: 06644001-f99e-4861-a17e-dad37c2d44f4
    
Generated RootId: hellopelle-06644001
```